demos/face_recognition_demo/python/README.md

```diff
@@ -11,7 +11,7 @@ The following pretrained models can be used:
 
 * `face-detection-retail-0004` and `face-detection-adas-0001`, to detect faces and predict their bounding boxes;
 * `landmarks-regression-retail-0009`, to predict face keypoints;
-* `face-reidentification-retail-0095` or `Sphereface`, to recognize persons.
+* `face-reidentification-retail-0095`, `Sphereface` or `face-recognition-resnet100-arcface-onnx`, to recognize persons.
```

Comment:
you forgot about https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/facenet-20180408-102900

Reply:
Done